src/pubsub/mod_pubsub.erl

```diff
@@ -1617,14 +1620,14 @@ adhoc_get_pending_parse_options(Host, #xmlel{name = <<"x">>} = XEl) ->
             end
     end;
 adhoc_get_pending_parse_options(_Host, XData) ->
-    ?INFO_MSG("Bad XForm: ~p", [XData]),
+    ?LOG_INFO(#{what => pubsub_bad_xform, exml_packet => XData}),
     {error, mongoose_xmpp_errors:bad_request()}.
 
 %% @doc <p>Send a subscription approval form to Owner for all pending
 %% subscriptions on Host and Node.</p>
 send_pending_auth_events(Request, Host, Node, Owner) ->
-    ?DEBUG("Sending pending auth events for ~s on ~s:~s",
-           [jid:to_binary(Owner), Host, Node]),
+    ?LOG_DEBUG(#{what => pubsub_sending_pending_auth_events,
```

Comment:
I think we would use `node` field as erlang node for each event (something that logger would emit). The same with pid, module, function, line, timestamp, when fields. So, basically, if you pass node now, it would be overwritten later. You also use function below, so it should be something action_function.


So, pubsub nodes should use `pubsub_node` field instead.